src/daemon/ca/status.rs

```diff
@@ -177,6 +190,30 @@ impl StatusStore {
             .await
     }
 
+    /// Adds a child with default status values if the child is missing
+    pub async fn set_child_default_if_missing(&self, ca: &Handle, child: &ChildHandle) -> KrillResult<()> {
+        self.update_ca_child_status(ca, child, |_status| {}).await
+    }
+
+    /// Remove a CA from the saved status
+    /// This should be called when the CA is removed from Krill, but note that if this is done for a CA which still exists
+    /// a new empty default status will be re-generated when it is accessed for this CA.
+    pub async fn remove_ca(&self, ca: &Handle) -> KrillResult<()> {
+        let mut cache = self.cache.write().await;
+
+        if !cache.contains_key(ca) {
+            Ok(()) // idempotent
+        } else {
+            let scope = ca.as_str();
+
+            self.store.drop_scope(scope)?; // idem potent; won't do anything if the scope does not exist.
```

Comment:
s/idem potent/idempotent/, but perhaps also redundant to say as "won't do anything" also implies it is idempotent.

Also, you could slightly simplify this fn by doing:
```
if cache.contains_key(ca) {
    ...
}
Ok(())
```

Reply:
Fixed the comment.

Left the logic as-is, to me it's more explicit this way the removal just won't do anything when there is nothing to remove, rather then throwing an error.